lib/srv/db/mysql/gcp.go

```diff
@@ -124,37 +126,37 @@ func (e *Engine) getGCPUserAndPassword(ctx context.Context, sessionCtx *common.S
 	}
 }
 
-func (e *Engine) getGCPIAMAuthToken(ctx context.Context, sessionCtx *common.Session) (string, error) {
-	e.Log.DebugContext(ctx, "Authenticating GCP MySQL with IAM auth.", "session", sessionCtx)
+func (c *connector) getGCPIAMAuthToken(ctx context.Context, dbUser string) (string, error) {
+	c.log.DebugContext(ctx, "Authenticating GCP MySQL with IAM auth.", "db_user", dbUser)
```

Comment:
I think we should keep `connector` in a single file, but perhaps these methods can belong to a different type.